integrations/pom.xml

```diff
@@ -253,7 +253,7 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>structs</artifactId>
-      <version>1.17</version>
+      <version>1.19</version>
```

Comment:
@casz This was the change made

Reply:
I have no issue with running `mvn install` inside IDEA did you remember to either import POM changes or enable auto update based on changes inside the POM?

Reply:
I refresh the dependencies everytime I make a change, so I think I should be fine.